Workflow/Sources/Workflow/Models/AnyWorkflow.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  AnyWorkflow.swift
+//  
+//
+//  Created by Tyler Thompson on 11/25/20.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import Foundation
+
+public class AnyWorkflow: LinkedList<FlowRepresentableMetadata> {
+    public typealias InstanceNode = LinkedList<AnyFlowRepresentable?>.Element
+    public typealias ArrayLiteralElement = AnyFlowRepresentable.Type
+    internal var instances = LinkedList<AnyFlowRepresentable?>()
+    internal var orchestrationResponder: AnyOrchestrationResponder?
+
+    public func applyOrchestrationResponder(_ orchestrationResponder: AnyOrchestrationResponder) {
+        self.orchestrationResponder = orchestrationResponder
+    }
+
+    @discardableResult public func launch(withLaunchStyle launchStyle: LaunchStyle = .default,
+                                          onFinish: ((Any?) -> Void)? = nil) -> LinkedList<AnyFlowRepresentable?>.Element? {
+        launch(passedArgs: .none, withLaunchStyle: launchStyle, onFinish: onFinish)
+    }
+
+    @discardableResult public func launch(with args: Any?,
```

Comment:
Is it possible to use the `Workflow` class and get a type-safe launch? The args would have to be the *first* FlowRepresentable.WorkflowInput value. It may be impossible or simply not reasonable, but it would be quite nice as a consumer I think.